src/main.js

```diff
@@ -1,29 +1,51 @@
 //function that checks if the env value is valid
 //value cannot start with a digit and must only contain alphanumeric characters or an underscore
 const checkValidationOfValues = envSpecString => {
+  const validValues = ['color','date','datetime-local','email','month','number','password','tel','text','time','url','week','default'];
```

Comment:
Two things here:

1. We should rename this to `validTypes`
2. We should remove `default` from the valid types. There is no `default` type in the provided link in https://github.com/sourcelair/env-spec-js/issues/3